perses/rjmc/geometry.py

```diff
@@ -624,23 +662,24 @@ def _normalize_torsion_proposal(self, atom, r, theta, bond_atom, angle_atom, tor
             xyzs[i], _ = self._autograd_itoc(bond_atom.idx, angle_atom.idx, torsion_atom.idx, r, theta, phi, positions, calculate_jacobian=False)
 
         #set up arrays for energies from angles and torsions
-        ub= np.zeros(n_divisions)
+        logq = np.zeros(n_divisions)
         for i, xyz in enumerate(xyzs):
-            ub_i = self._torsion_and_angle_potential(xyz, atom, positions, involved_angles, involved_torsions, beta)
-            if np.isnan(ub_i):
-                ub_i = np.inf
-            ub[i]+=ub_i
+            logq_i = self._torsion_and_angle_logq(xyz, atom, positions, involved_angles, involved_torsions, beta)
+            if np.isnan(logq_i):
```

Comment:
`logq` should never be `NaN`.  Let's eliminate this, since we want to flag this as a bug if we run into `NaN`.